website/profile/views.py

```diff
@@ -374,6 +374,43 @@ def user_notifications(auth, **kwargs):
         'mailing_lists': auth.user.mailing_lists
     }
 
+@dev_only
```

Comment:
Please add a `TODO` comment here to remove `dev_only` when APIv2 is in production.